vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
Hmm, this all seems a bit brittle to me.  It also seems unclear how a dataset is supposed to choose the enumeration value (i.e. so that different datasets would be using the same value).  For example, what is the difference between city, local, locality, and municipality (and county in the case of a City and County like San Francisco)?  Or utility, water, other?  Or council and ward?  And what about when there are multiple "other" values (e.g. BART and the Board of Equalization examples I mentioned above)?

FWIW, I did find what seem to be at least three types of judicial districts in California, each with its own numbering sequence (presumably all applicable in retention elections). The California courts of appeal has six districts within the state. Los Angeles County Superior Court of California has thirteen judicial districts.  And the Fourth District of the California Court of Appeal has three districts within that.

For "school," note that there are overlapping districts here as well (e.g. Community College districts and the school districts for lower levels like elementary and high school).

Also, just to be clear, my original suggestion wasn't to invent a new identification scheme, but to introduce a parent object that would serve to group the districts within a numbering sequence.  It is agnostic to the particular identification scheme: it doesn't require one but could still be used in conjunction with one (and in conjunction with 1622.2).